db/models/articles.js

```diff
@@ -13,7 +21,7 @@ exports.fetchArticles = topic => {
 	FROM articles LEFT JOIN comments ON articles.article_id = comments.article_id 
 	${whereClause}
 	GROUP BY articles.article_id
-	ORDER BY articles.created_at DESC;`;
+	${orderClause};`;
 
 	return db.query(queryStr, queryParams).then(async results => {
```

Comment:
there's a mixture of .then and async/await here. 
Best practice is to stick with one